holotrack/Tracking/FaceTracker.cs

```diff
@@ -104,24 +111,37 @@ private void trackerLoop(CancellationToken cancellationToken)
 
                 if (camera?.CaptureData == null)
                     continue;
+                    
+                // init stopwatch
+                var stopwatch = new Stopwatch();
```

Comment:
hide this under a preprocessor directive, same goes with all stopwatch methods. Since you mentioned it's for debugging you can hide it under preprocessor directives instead of removing it entirely.
```suggestion
                #if DEBUG
                var stopwatch = new Stopwatch();
                
                stopwatch.Start();
                #endif 
```